src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -188,21 +188,22 @@ class GoogleExtensions(val directoryDAO: DirectoryDAO, val accessPolicyDAO: Acce
   def createUserPetServiceAccount(user: WorkbenchUser, project: GoogleProject): Future[PetServiceAccount] = {
     val (petSaID, petSaDisplayName) = toPetSAFromUser(user)
 
-    directoryDAO.loadPetServiceAccount(PetServiceAccountId(user.id, project)).flatMap {
-      case Some(pet) => Future.successful(pet)
-      case None =>
-        // First find or create the service account in Google, which generates a unique id and email
-        val petSA = googleIamDAO.getOrCreateServiceAccount(project, petSaID, petSaDisplayName)
-        petSA.flatMap { serviceAccount =>
-          // Set up the service account with the necessary permissions
-          val petServiceAccount = PetServiceAccount(PetServiceAccountId(user.id, project), serviceAccount)
+    // First find or create the service account in Google, which generates a unique id and email
+    val petSA = googleIamDAO.getOrCreateServiceAccount(project, petSaID, petSaDisplayName)
+    petSA.flatMap { serviceAccount =>
+      // Set up the service account with the necessary permissions
+      val petServiceAccount = PetServiceAccount(PetServiceAccountId(user.id, project), serviceAccount)
```

Comment:
This is thrown away if it's in OpenDJ so we should move that to the `None` below